### PR TITLE
Generate documentation ...

### PR DESCRIPTION
**Pull request by waitan2018**

### CODE REVIEW

**Review by GJJ121785021 — APPROVED** [review]

File: django_simple_api/templates/redoc.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
 </head>
 
 <body>
-<redoc spec-url='./docs'></redoc>
+<redoc spec-url='./get_docs/'></redoc>
```
Comment (abersheeran) [inline django_simple_api/templates/redoc.html:28]: Change to "get-docs"

File: django_simple_api/templates/swagger.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
 <script src="https://cdn.jsdelivr.net/npm/swagger-ui-dist@3.30.0/swagger-ui-bundle.js"></script>
 <script>
     const ui = SwaggerUIBundle({
-        url: './docs',
+        url: './get_docs/',
```
Comment (abersheeran) [inline django_simple_api/templates/swagger.html:14]: too.

File: django_simple_api/urls.py [hunk 3]
```diff
@@ -3,5 +3,6 @@
 from . import views
 
 urlpatterns = [
-    path("get-docs/", views.get_docs, name="get-docs"),
+    path("", views.docs, name="docs"),
+    path("get_docs/", views.get_docs, name="get_docs"),
```
Comment (abersheeran) [inline django_simple_api/urls.py:7]: too.

**Review by abersheeran — CHANGES_REQUESTED** [review]

File: django_simple_api/templates/swagger.html (hunk 2 above)
Comment (MeGustas-5427) [reply django_simple_api/templates/swagger.html:14]: too. +1

File: django_simple_api/urls.py (hunk 3 above)
Comment (MeGustas-5427) [reply django_simple_api/urls.py:7]: too. +1

**Review by GJJ121785021 — DISMISSED** [review]

File: django_simple_api/templates/redoc.html (hunk 1 above)
Comment (MeGustas-5427) [reply django_simple_api/templates/redoc.html:28]: Change to "get-docs" +1

**Review by GJJ121785021 — APPROVED** [review]

**Review by abersheeran — APPROVED** [review]
Merge

**Pull request merged**